src/components/buttons/AddIdeaButton/AddIdeaButton.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import './AddIdeaButton.css';
 import Parse from 'parse';
+import { IoIosAddCircleOutline } from 'react-icons/io'
 
 const AddIdeaButton = ({
     text,
-    UserIcon,
-    trigger,
+    setPopup,
     setIdeaId
 }) => {
 
     function handleClickPopup() {
-        trigger(true)
+        console.log("bt clicked!")
```

Comment:
I assume you have used `console.log("bt clicked!")` for debugging. I do the exact same, but before we hand in the code/exam, we should probably just remember to remove them 😅